airbyte-integrations/connector-templates/source-generic/acceptance-test-docker.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+
+# Build latest connector image
+docker build . -t $(cat acceptance-test-config.yml | grep "connector_image" | head -n 1 | cut -d: -f2-)
```

Comment:
[comment]is there a `jq` for yaml?  That might be nice here...

Reply:
there is, it's called (creatively enough) `yq`. I didn't add it here because I didn't want to use non-standard unix tools in this script to minimize chances of random issues on developer (especially community dev) laptops